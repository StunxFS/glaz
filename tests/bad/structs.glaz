struct Account {
  name: str,
  age: i32
}

fn main() {
  _ = Account{
    name: "Epa!",
    age: 10,
    ff: 0x1A
  };
  _ = Account{
    name: "Epa!",
    age: true
  };
}
