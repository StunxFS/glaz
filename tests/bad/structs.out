tests/bad/structs.glaz:10:6: error: struct `Account` has no field `ff`
     | 
  10 |     ff: 0x1A
     |     ^^
tests/bad/structs.glaz:14:11: error: expected type `i32`, found `bool`
     | 
  14 |     age: true
     |          ^^^^
     = note: in field `age` of struct `Account`
glaz error: could not compile `structs`, aborting due to 2 previous errors.
